tasks/config.yml

```diff
@@ -44,6 +44,14 @@
   become_user: "{{ superset_user }}"
   command: "{{ superset_virtualenv_path }}/bin/superset init"
 
+- name: Copy uwsgi.ini
+  template:
+    src: uwsgi.ini.j2
+    dest: "{{ superset_virtualenv_path }}/uwsgi.ini"
```

Comment:
Can we copy the ini file to `superset_codebase_path`?

Reply:
Would it make sense to have it under site-packages? This is superset base path https://github.com/onaio/ansible-superset/blob/master/defaults/main.yml#L203